static/app/components/sidebar/sidebarItem.tsx

```diff
@@ -311,9 +319,28 @@ function SidebarItem({
 }
 
 export function isItemActive(
-  item: Pick<SidebarItemProps, 'to' | 'label'>,
+  item: Pick<SidebarItemProps, 'to' | 'label' | 'search'>,
   exact?: boolean
 ): boolean {
+  // issue subnav matches against issue groups in search params
+  if (location.pathname.startsWith('/issues/') && item?.to?.includes('/issues/')) {
```

Comment:
Will this only be relevant for issues subnav?
Will we ever need to implement logic for other routes?

wondering about scalability and how we might structure this or break this out to be more composable 🤷